docs/node/activities.md

```diff
@@ -23,32 +24,81 @@ Note that we only import the type of our activities, the TypeScript compiler wil
+| `heartbeatTimeout`       | A best practice to set for long-running activities.                                                                                                                                                                                                                              |
+| `retry`                  | [RetryOptions](https://nodejs.temporal.io/api/interfaces/worker.RetryOptions) that define how activity is retried in case of failure. If this is not set, then the server-defined default activity retry policy will be used. To ensure zero retries, set maximum attempts to 1. |
+| `activityId`             | Identifier to use for tracking the activity in Workflow history. The `activityId` can be accessed by the activity function. Does not need to be unique. Defaults to an incremental sequence number.                                                                              |
+| `taskQueue`              | Task queue name. defaults to current worker task queue.                                                                                                                                                                                                                          |
 
-A Workflow can request to cancel an Activity by cancelling its containing [cancellation scope](/docs/node/cancellation-scopes).
+You can specify timeouts as number of milliseconds, or a string parsed to number of milliseconds by the [`ms`](https://www.npmjs.com/package/ms) package:
 
-Activities may be cancelled only if they emit heartbeats.
-There are 2 ways to handle Activity cancellation:
+```
+ms('2 days')   // 172800000
+ms('1d')       // 86400000
+ms('10h')      // 36000000
+ms('2.5 hrs')  // 9000000
+ms('2h')       // 7200000
+ms('2 hours')  // 7200000
+ms('1 minute') // 60000
+ms('1m')       // 60000
+ms('5s')       // 5000
+ms('1y')       // 31557600000
+ms('100')      // 100
+```
 
-1. Await on [`Context.current().cancelled`](https://nodejs.temporal.io/api/classes/activity.context#cancelled)
-1. Pass the context's abort signal at [`Context.current().cancellationSignal`](https://nodejs.temporal.io/api/classes/activity.context#cancelled) to a library that supports it like `fetch`
+To better understand Activity timeouts, refer to our blogpost on the [4 Types of Activity timeouts](https://docs.temporal.io/blog/activity-timeouts).
 
-[`heartbeat()`](https://nodejs.temporal.io/api/classes/activity.context/#heartbeat) in the Node.js SDK is a background operation and does not propagate errors to the caller, such as when the scheduling Workflow has already completed or the Activity has been closed by the server (due to timeout for instance). These errors are translated into cancellation and can be handled using the methods above.
+## How to register an Activity
```

Comment:
Up to you at this stage, but this seems like a clearly scoped piece of "how to" content
Also, this does exist, perhaps you can make use of it:
https://docs.temporal.io/docs/content/how-to-develop-a-worker-program-in-node/

Reply:
ah excellent! will merge, ty for the catch

Reply:
@sw-yx @flossypurse note that the how to page is out-of-date now.

We've been rapidly changing the SDK's API in the last few weeks in prep for beta and I wasn't even aware this content was there, otherwise I'd have updated it.
It'd be nice if all of the ~~Node~~ TS SDK content was gathered in a single directory so it'd be easier to locate.

Reply:
ya no worries about this page, i will update it